experiment/run_experiment.py

```diff
@@ -371,12 +389,12 @@ def main():
     args = parser.parse_args()
 
     if not args.fuzzers and not args.fuzzer_configs:
-        fuzzers = get_all_fuzzers()
+        fuzzer_configs = get_all_fuzzer_configs()
     else:
-        fuzzers = args.fuzzers
+        fuzzer_configs = args.fuzzer_configs
```

Comment:
I guess it doesn't need to come right now but do you think we should even have a way to specify configs (aka variants) from outside the repo if we have a way to support it within repo? We should probably get rid of this feature/command line option.